native/src/routes/news/containers/_tests_/NewsContainer.spec.js

```diff
@@ -175,21 +175,6 @@ describe('News', () => {
     expectLoadingIndicator(state)
   })
 
-  it('should display loading indicator if switching languages lasts long enough', () => {
-    const state: StateType = prepareState(successfulRouteState, { switchingLanguage: true })
-    expectLoadingIndicator(state)
```

Comment:
Why were these cases removed?

Reply:
Because this is no longer handled in the NewsContainer but in the withPayloadProvider